app/src/main/java/org/oppia/android/app/translation/AppLanguageResourceHandler.kt

```diff
@@ -132,6 +132,7 @@ class AppLanguageResourceHandler @Inject constructor(
   /** See [OppiaLocale.DisplayLocale.getLayoutDirection]. */
   fun getLayoutDirection(): Int = getDisplayLocale().getLayoutDirection()
 
-  private fun getDisplayLocale(): OppiaLocale.DisplayLocale =
+  /** Returns the current [OppiaLocale.DisplayLocale] used for resource processing. */
+  fun getDisplayLocale(): OppiaLocale.DisplayLocale =
```

Comment:
Could we instead expose the functionality needed rather than the locale (like we do for other locale functions above)?